src/test/unit/rc_controls_unittest.cc

```diff
@@ -48,6 +48,8 @@ extern "C" {
     #include "fc/rc_adjustments.h"
 
     #include "fc/rc_controls.h"
+
+    #include "scheduler/scheduler.h"
```

Comment:
Adding this file was not necessary I think

Reply:
i cretaed testing enviroment under cygwin and it failed without this include

Reply:
That's unfortunate, you could stub it with ints though

Reply:
its not stubed in arming_prevention_unitest so why do it here ?